libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/controller/CameraStateController.kt

```diff
@@ -17,16 +18,24 @@ internal class CameraStateController(
     }
 
     override fun process(state: State, action: Action): State {
-        if (action is CameraAction) {
-            return state.copy(camera = processCameraAction(state.camera, action))
+        return when (action) {
+            is Command.RecenterCamera ->
```

Comment:
Will you add tests for tests for these new cases? `CameraStateControllerTest` is unchanged